### PR TITLE
add ccbar assym basis

### PR DESCRIPTION
**Pull request by RoyStegeman**
Addresses #1655

### CODE REVIEW

**Review by tgiani — APPROVED** [review]

File: validphys2/src/validphys/pdfbases.py [hunk 1]
```diff
@@ -753,6 +756,17 @@ def fitbasis_to_NN31IC(flav_info, fitbasis):
...
+        sng = {'u': 1, 'ubar': 1, 'd': 1, 'dbar': 1, 's': 1, 'sbar': 1, 'c': 1, 'cbar': 1, 'g': 0 }
+        v = {'u': 1, 'ubar': -1, 'd': 1, 'dbar': -1, 's': 1, 'sbar': -1, 'c': 1, 'cbar': -1, 'g': 0 }
+        v3 = {'u': 1, 'ubar': -1, 'd': -1, 'dbar': 1, 's': 0, 'sbar': 0, 'c': 0, 'cbar': 0, 'g': 0 }
+        v8 = {'u': 1, 'ubar': -1, 'd': 1, 'dbar': -1, 's': -2, 'sbar': 2, 'c': 0, 'cbar': 0, 'g': 0 }
+        t3 = {'u': 1, 'ubar': 1, 'd': -1, 'dbar': -1, 's': 0, 'sbar': 0, 'c': 0, 'cbar': 0, 'g': 0 }
+        t8 = {'u': 1, 'ubar': 1, 'd': 1, 'dbar': 1, 's': -2, 'sbar': -2, 'c': 0, 'cbar': 0, 'g': 0 }
+        cp = {'u': 0, 'ubar': 0, 'd': 0, 'dbar': 0, 's': 0, 'sbar': 0, 'c': 1, 'cbar': 1, 'g': 0 }
+        g = {'u': 0, 'ubar': 0, 'd': 0, 'dbar': 0, 's': 0, 'sbar': 0, 'c': 0, 'cbar': 0, 'g': 1 }
+        v15 = {'u': 1, 'ubar': -1, 'd': 1, 'dbar': -1, 's': 1, 'sbar': -1, 'c': -3, 'cbar': 3, 'g': 0 }
+
```
Comment (scarlehoff) [inline validphys2/src/validphys/pdfbases.py:769]: Seems correct to me, but we should really rethink this function sooner or later...

**Review by scarlehoff — APPROVED** [review]

**Comment by tgiani** [comment]
seems correct to me as well

File: validphys2/src/validphys/pdfbases.py (hunk 1 above)
Comment (RoyStegeman) [reply validphys2/src/validphys/pdfbases.py:769]: True, it's not exactly the cleanest bit of code

**Comment by RoyStegeman** [comment]
Thanks, I'll wait for the tests to pass and then merge

**Pull request merged**